e2e_tests/resources/strings.py

```diff
@@ -13,7 +13,7 @@
 AZUREML_SERVICE = "tre-service-azureml"
 DEVTESTLABS_SERVICE = "tre-service-devtestlabs"
 GUACAMOLE_SERVICE = "tre-service-guacamole"
-INNEREYE_SERVICE = "tre-service-innereye"
```

Comment:
Why delete INNEREYE_SERVICE when you use it /workspaces/AzureTRE/e2e_tests/test_workspace_service_creation.py

Reply:
It's back.